.env

```diff
@@ -1,3 +1,7 @@
 BUILDPACK_URL=https://github.com/heroku/heroku-buildpack-python.git
 
-SENTRY_DSN=https://<key>:<secret>@app.getsentry.com/<project>
\ No newline at end of file
+SENTRY_DSN=https://<key>:<secret>@app.getsentry.com/<project>
+
+
+SUPPORT_URL = https://docs.google.com/forms/d/e/1FAIpQLSdkfLU2yi2S1_7D27Z0I1TumkWy5brlam809Od9cc6CnXGA-A/viewform
+INQUIRY_EMAIL = civicsignal@codeforafrica.org
```

Comment:
Do these need to be here? Can't they be sent on the prod instance itself?

Reply:
Yes, those should be sent to production .env file was already in the repo. Unless we need to remove the file now from the repository.